tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -0,0 +1,143 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation     Test 23-03 - VCH Create
+Resource          ../../resources/Util.robot
+Resource          ../../resources/Group23-VIC-Machine-Service-Util.robot
+Suite Setup       Setup
+Suite Teardown    Teardown
+Default Tags
+
+
+*** Keywords ***
+Setup
+    Start VIC Machine Server
+    Set Test Environment Variables
+
+
+Teardown
+    Terminate All Processes    kill=True
+
+
+Create VCH
+    [Arguments]    ${data}
+    Post Path Under Target    vch    ${data}
+
+
+Count VCHs
+    Get Path Under Target    vch
+    ${count}=    Run    echo '${OUTPUT}' | jq -r '.vchs | length'
+    ${count}=    Convert To Integer    ${count}
+    [Return]    ${count}
+
+
+*** Test Cases ***
+Create minimal VCH
+    ${before}=    Count VCHs
```

Comment:
I'm not sure if counting number of VCHs is required here but it may create issue when we have these tests running concurrently. Just FYI.

Reply:
Good point. I'll remove the counting; I think being able to inspect it is sufficient proof that the VCH was properly created.